norbit-webapp/components/Map/Map.tsx

```diff
@@ -0,0 +1,69 @@
+import { Map } from 'leaflet';
+import React, { useEffect, useRef, useState } from 'react';
+import { TamplifyInstance } from '@/dashboard/Dashboard';
+import { useSubscribeToTopics } from 'utils/useSubscribeToTopic';
+import { set } from 'cypress/types/lodash';
+import { MapData } from './types';
+
+
+// Deferred import for Leaflet
+let L: typeof import('leaflet');
+if (typeof window !== 'undefined') {
+  L = require('leaflet');
+}
+
+
+const LeafletMap: React.FC<{ amplifyInstance: TamplifyInstance | null }> = ({ amplifyInstance }) => {
```

Comment:
In line with our other component naming (and repo conventions), this should be named MapComponent, and the file and folder should also be called MapComponent.

Reply:
makes sense, well done!